test/unit/spec/nbextensions/appCell2/widgets/appCellWidget-spec.js

```diff
@@ -429,24 +535,82 @@ define([
                 });
 
                 it('can start in an internal error state', async function () {
+                    setUpCellInState(this, 'INTERNAL_ERROR');
+                    await startRunningCell(this);
+
+                    expect(this.appCellWidgetInstance.__fsm().getCurrentState().state).toEqual(
+                        fsmState.INTERNAL_ERROR
+                    );
+                    // the reset button should be active
+                    const resetButton = this.kbaseNode.querySelector(selectors.reset);
+                    expect(resetButton).not.toHaveClass('hidden');
+                    expect(resetButton).not.toHaveClass('disabled');
```

Comment:
It should be possible to test the tab content, but there's something weird going on with the app cell's tab mechanism and it seems as though you can only get them to show up by manually clicking on them (rather than using `button.click()`). Ideally (if I have time this week) I will switch the app cell over to using the `CellTabs` module that the bulk import cell uses.